src/xbps/repositories/official/index.md

```diff
@@ -3,11 +3,23 @@
 Void provides other official repositories, which are maintained by the Void
 project, but not installed in the default configuration:
 
-- [debug](./debug.md): contains debugging symbols for packages
-- [multilib](./multilib.md): contains 32-bit libraries for 64-bit systems (glibc
-   only)
-- [multilib/nonfree](./multilib.md): contains non-free multilib packages
-- [nonfree](./nonfree.md): contains software packages with non-free licenses
-
-The next sections provide information on installing repositories, and summaries
-of the different repositories offered.
+| Package			| Description						| Link					|
+|-------------------------------|-------------------------------------------------------|---------------------------------------|
+| `void-repo-multilib`		| 32-bit compatibility packages 			| [multilib](./multilib.md)		|
+| `void-repo-mulitlib-nonfree`	| 32-bit compatibility packages with nonfree components | [multilib-nonfree](./multilib.md)	|
+| `void-repo-nonfree`		| packages with unfree licenses 			| [nonfree](./nonfree.md)			|
+| `void-repo-debug`		| debug symbols for packages 				| [debug](./debug.md)			|
+
+The next sections provide summaries of the different repositories offered.
+
+## Installing
+
+Those repositories may be installed via the packages from the package column. 
+For example, to install the `nonfree` repository, install the package `void-repo-nonfree`:
+
+```
+# xbps-install -S void-repo-nonfree
```

Comment:
```suggestion
# xbps-install void-repo-nonfree
```

The -S would have to happen after installation, not before. If the reason for the -S is that the user should have synced current package data, then I think this goes without saying.

Reply:
It seems to be common practice to have that -S flag there.

Reply:
I think nilium is technically right on this one.

Reply:
Hold off on changing anything here while I check on IRC.

Reply:
This is a separate issue we don't need to deal with in this commit.